qiskit/algorithms/amplitude_amplifiers/grover.py

```diff
@@ -58,10 +58,10 @@ class Grover(AmplitudeAmplifier):
     The details of how :math:`S_f` works are unimportant to the algorithm; Grover's
     search algorithm treats the oracle as a black box.
 
-    This class supports oracles in form of :class:`~qiskit.QuantumCircuit`
+    This class supports oracles in form of :class:`~qiskit.QuantumCircuit`.
 
-    With oracle at hand, Grover's Search constructs the Grover operator to amplify the amplitudes
-    of the good states:
+    With the oracle at hand, Grover's Search constructs the Grover operator to amplify the
```

Comment:
```suggestion
    With the given oracle, Grover's Search constructs the Grover operator to amplify the
```

Reply:
`at hand` might be harder for some people to understand its means especially if english if not your native language - I think this might be a little easier

Reply:
I took off the automerge label since I just did these as comments not requested change